src/main/java/use_case/TaskGetter.java

```diff
@@ -2,26 +2,28 @@
 
 import main.java.entity.Task;
 import main.java.entity.TodoList;
+import main.java.entity_gateway.TaskReader;
+import main.java.entity_gateway.TodoListManager;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
-public class TaskGetter {
-    /**
-     * @param todoData (stores a todolist which stores a list of tasks)
-     * @return a list of tasks organized in map format, with
-     * "name", "deadline", "subtasks", and "completed" as keys
-     */
-    public List<HashMap<String, String>> getTasks(AccessTodoData todoData) {
-        TodoList todoList = todoData.getTodoList();
-        List<HashMap<String, String>> task_data = new ArrayList<>();
-        for(Task task : todoList.getTasks()) {
-            task_data.add(getTask(task));
-        }
+public class TaskGetter implements TodoListDisplayBoundary {
 
-        return task_data;
+    private final TodoListManager todoListManager;
+    private final TodoListPresenter taskPresenter;
+
+    public TaskGetter(TodoListManager todoListManager, TodoListPresenter taskPresenter) {
+        this.todoListManager = todoListManager;
+        this.taskPresenter = taskPresenter;
+    }
+
+    public TodoListsInfo getTasks() {
```

Comment:
What is this method used for?

Reply:
The `ApplicationDriver` wanted it to display all the current tasks. I wanted to delete it because it's extremely redundant with the `presentAllTodoLists()` however I felt bad about deleting a bunch of stuff at once and wanted to do it in smaller PRs where it's more visible instead of hidden in this big one.